resources/views/themes/default1/front/checkout.blade.php

```diff
@@ -434,4 +434,43 @@
     
 </div>
 @endif
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+<script>
+  $('#checkoutsubmitform').submit(function(){
+     $("#proceed").html("<i class='fa fa-circle-o-notch fa-spin fa-1x fa-fw'></i>Please Wait...")
+    $("#proceed").prop('disabled', true);
+
+  });
+     $(document).ready(function(){
+            var finalPrice = $('#total-price').val();
+            $("#rzp_selected").click(function(){
+            
+            var processingFee = $(this).attr('data-currency');
+            var totalPrice = finalPrice;
+            $('#fee').hide();
+            $.ajax({
```

Comment:
do we not have a csrf check in our backend?

Reply:
added now